catalog/transformer/catalog-transformer-csv-queryresponse/src/main/java/ddf/catalog/transformer/csv/CsvMetacardTransformer.java

```diff
@@ -43,12 +48,21 @@ public BinaryContent transform(Metacard metacard, Map<String, Serializable> argu
 
     Map<String, String> aliases =
         (Map<String, String>) arguments.getOrDefault("aliases", new HashMap<>());
-
+    String attributeString =
+        arguments.get("columnOrder") != null ? (String) arguments.get("columnOrder") : "";
+    List<String> attributes = new ArrayList<String>(Arrays.asList((attributeString).split(",")));
```

Comment:
any reason these all need to be `new ArrayList`? I think it should already be returning a list of strings. Same question for the two below

Reply:
I think two of them can be changed, the `metacard.getMetacardType().getAttributeDescriptors()` returns a `Set` so I'll probably leave that as is. Alternatively I could instantiate `allAttributes` as a `Set`, but i'd just have to change it to a list when instantiating `descriptors` so I'll leave it as-is. I can change the other two though